src/FFMpeg/FFMpeg.php

```diff
@@ -93,7 +93,14 @@ public function open($pathfile)
             throw new RuntimeException(sprintf('Unable to probe "%s".', $pathfile));
         }
 
-        if (0 < count($streams->videos())) {
+        $hasVideo = false;
+        foreach ($streams->videos() as $videoStream) {
+            $videoDisposition = $videoStream->get('disposition');
+            if ($videoDisposition['attached_pic'] !== 1) {
```

Comment:
I find this a little bit strange, should we not use instead a mime_type check on pathfile to check if its a video or audio file. I could think that there are videos without a attached_pic disposition so I would not depend on that.
Also it feels for me the false position to fix it as $streams->videos() should not return none video file atleast I would not expect that.

@romainneutron do you remember why this was [done before](https://github.com/PHP-FFMpeg/PHP-FFMpeg/commit/ad3a5af623541d1f200aa29a68f90c191140e2a0) over `count($streams->videos())` instead of checking for mimetype?

Reply:
If it is possible to reply for @romainneutron yet due to the current circumstances, I'd be happy to bring this a little bit forward. :)